scripts/scil_connectivity_math.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+import os
+
+import numpy as np
+
+from scilpy.image.operations import (get_array_ops, get_operations_doc)
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_outputs_exist)
+from scilpy.utils.util import is_float
+
+OPERATIONS = get_array_ops()
+
+DESCRIPTION = """
+Performs an operation on a list of matrices. The supported operations are
+listed below.
+
+Some operations such as multiplication or addition accept float value as
+parameters instead of matrices.
+> scil_connectivity_math.py multiplication mat.npy 10 mult_10.npy
+"""
+
+ADDED_DOC = get_operations_doc(OPERATIONS).replace('images', 'matrices')
+ADDED_DOC = ADDED_DOC.replace('image', 'matrix')
+ADDED_DOC = ADDED_DOC.replace('IMG', 'MAT')
+DESCRIPTION += ADDED_DOC
```

Comment:
Did you try to do `__doc__ + ADDED_DOC`  in `description=..` ?

Reply:
Does not work